### PR TITLE
README.md 'Thank you' and Resource links - update

### PR DESCRIPTION
**Pull request by Danielcalza**
The first attempt at a pull request.

### CODE REVIEW

File: materials/index.md [hunk 1]
```diff
@@ -5,8 +5,144 @@ title: Lecture Materials
 
 ### Agroecology IST
 
+<<<<<<< HEAD
```
Comment (brymz) [inline materials/index.md:8]: Remove this line.
Something funky happened with the merge that we aborted during the lab meeting..

File: materials/index.md [hunk 2]
```diff
@@ -5,8 +5,144 @@ title: Lecture Materials
...
 - [Introductions](introductions)
 - [Why Agroecology?](why-agroecology)
 - [A Brief History](brief-history)
 - [Principles from Ecology](ecology-principles)
-- [Extension Work Groups, Initiatives, and Indicators](extension-impacts)
\ No newline at end of file
+- [Extension Work Groups, Initiatives, and Indicators](extension-impacts)
+=======
```
Comment (brymz) [inline materials/index.md:14]: Remove all these lines to the end.

**Comment by brymz** [comment]
Daniel,
I'm looking deeper at this. See me before you make any changes.
~Z

**Comment by Danielcalza** [comment]
Problems from an errant merge.

**Pull request closed without merging**